src/vstest.console/InProcessVsTestConsoleWrapper.cs

```diff
@@ -62,7 +65,29 @@ public InProcessVsTestConsoleWrapper(ConsoleParameters consoleParameters)
         ITestRequestManager? testRequestManager,
         Executor executor,
         ITestPlatformEventSource testPlatformEventSource)
+        : this(
```

Comment:
Is there a good reason to add another internal constructor when there already is one?

Reply:
Not really, I was being lazy and I didn't want to change all the tests, but for clarity we should have only 1 internal constructor, I agree. Making the change now 👍